subjects/devops/credentials_searches/README.md

```diff
@@ -0,0 +1,60 @@
+## credentials_searches
+
+### Instructions
+
+Create a file `credentials_searches.py` that contains a function `credentials_search` which takes no parameters and searches for the keys `password` and `secret` in a file called `logs.json`.
+
+- If both keys are found, the function should create a new json file named 'credentials.json' and save the values of those keys in it.
+
+- If only one of the keys is found, the function should create a new json file named 'credentials.json' and save the value of that key in it.
+
+- If neither key is found, the function should not create the 'credentials.json' file.
+
+- If the 'logs.json' file is empty or is not a json file, the function should not create the 'credentials.json' file.
+
+- If the logs.json file does not exist, the function should not create the 'credentials.json' file.
```

Comment:
to be consistent with the resto of the subject, I would put `logs.json` in between single quotes: `'logs.json'`

Reply:
Thanks. Done.